frontend/src/components/feature/chat/ChatMessage/MessageActions/MessageActions.tsx

```diff
@@ -10,22 +10,33 @@ import { toast } from 'sonner'
 import { getErrorMessage } from '@/components/layout/AlertBanner/ErrorBanner'
 import { AiOutlineEdit } from 'react-icons/ai'
 import { LuForward, LuReply } from 'react-icons/lu'
+import { MdOutlineEmojiEmotions } from "react-icons/md";
+import { ReactionObject } from '../MessageReactions'
 
 export interface MessageContextMenuProps {
     message?: Message | null,
     onDelete: VoidFunction
     onEdit: VoidFunction,
     onReply: VoidFunction,
-    onForward: VoidFunction
+    onForward: VoidFunction,
 }
 
-export const MessageContextMenu = ({ message, onDelete, onEdit, onReply, onForward }: MessageContextMenuProps) => {
+interface MessageReactionAnalyticsProps extends MessageContextMenuProps {
+    reactions: ReactionObject[],
+    onClose: VoidFunction,
+    isOpen: boolean,
+    open: VoidFunction
+}
+
+export const MessageContextMenu = ({ message, onDelete, onEdit, onReply, onForward, ...reactionProps }: MessageReactionAnalyticsProps) => {
```

Comment:
Why do we need to change the type here? The component already gets the message object which has the reactions in it. Also, the callback should be similar to the other ones - just add a new onViewReaction callback function here.